docs/getting_started.md

```diff
@@ -4,21 +4,23 @@ We want to make it as easy as possible for you to start engaging with Suzieq. We
 
 ### As a Docker Container
 
-To get started seeing the kind of analysis Suzieq provides today, we have a demo container that has data in including the the iamge.
-
+To get started seeing the kind of analysis Suzieq provides today, we have a demo container that has data in including the the image.
```

Comment:
Can you remove the second "the" please i..e change "including the the image" to "including the image"?